src/Compilers/Core/Portable/CommandLine/CommandLineArguments.cs

```diff
@@ -499,10 +499,10 @@ internal void ResolveAnalyzersFromArguments(
                 switch (e.ErrorCode)
                 {
                     case AnalyzerLoadFailureEventArgs.FailureErrorCode.UnableToLoadAnalyzer:
-                        diagnostic = new DiagnosticInfo(messageProvider, messageProvider.WRN_UnableToLoadAnalyzer, analyzerReference.FullPath, e.Message);
+                        diagnostic = new DiagnosticInfo(messageProvider, messageProvider.WRN_UnableToLoadAnalyzer, analyzerReference.FullPath, e.Exception?.ToString() ?? e.Message);
```

Comment:
I don't know why we're not doing this today, but it really makes it hard to troubleshoot issues. @chsienki or @jaredpar, any chance either of you remember? Is this an msbuild multiple lines thing?

Reply:
I've done something similar to exceptions thrown by analyzers in https://github.com/dotnet/roslyn/pull/70031. I think I just haven't realized there is another place where exceptions are reported as diagnostics, otherwise I'd handle it too.

Reply:
> Is this an msbuild multiple lines thing?

I _suspect_ this will cause issues with msbuild in some edge cases. As to _why_ I'm guessing that it was a bit of an oversight when these diagnostics were created vs. an intentional decision.